solutions/kueue-admission-check/setup-env.sh

```diff
@@ -0,0 +1,114 @@
+kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/v0.7.1/manifests.yaml --context ${c2ctx}
+kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/v0.7.1/manifests.yaml --context ${c3ctx}
+
+echo "Install Jobset for MultiKueue"
+kubectl apply --server-side -f https://github.com/kubernetes-sigs/jobset/releases/download/v0.5.2/manifests.yaml --context ${hubctx}
+kubectl apply --server-side -f https://github.com/kubernetes-sigs/jobset/releases/download/v0.5.2/manifests.yaml --context ${c1ctx}
+kubectl apply --server-side -f https://github.com/kubernetes-sigs/jobset/releases/download/v0.5.2/manifests.yaml --context ${c2ctx}
+kubectl apply --server-side -f https://github.com/kubernetes-sigs/jobset/releases/download/v0.5.2/manifests.yaml --context ${c3ctx}
+
+kubectl config use-context ${hubctx}
+
+echo "Patch permission"
+kubectl patch clusterrole cluster-manager --type='json' -p "$(cat env/patch-clusterrole.json)"
+
+echo "Patch image"
+kubectl patch deployment cluster-manager -n open-cluster-management --type=json -p='[
+  {"op": "replace", "path": "/spec/template/spec/containers/0/image", "value": "quay.io/haoqing/registration-operator:latest"},
+  {"op": "replace", "path": "/spec/template/spec/containers/0/imagePullPolicy", "value": "Always"}
+]'
+kubectl patch clustermanager cluster-manager --type=json -p='[{"op": "replace", "path": "/spec/registrationImagePullSpec", "value": "quay.io/haoqing/registration:latest"}]'
+kubectl patch clustermanager cluster-manager --type=json -p='[{"op": "replace", "path": "/spec/placementImagePullSpec", "value": "quay.io/haoqing/placement:latest"}]'
+
+echo "Install CRDs"
+kubectl create -f env/multicluster.x-k8s.io_authtokenrequests.yaml
+kubectl create -f env/multicluster.x-k8s.io_clusterprofiles.yaml
+
+echo "Install managed-serviceaccount"
+cd /path/to/managed-serviceaccount  # TODO: Replace here with your actual path.
+helm uninstall -n open-cluster-management-addon managed-serviceaccount || true
+helm install \
+   -n open-cluster-management-addon --create-namespace \
+   managed-serviceaccount charts/managed-serviceaccount/ \
+   --set tag=latest \
+   --set featureGates.ephemeralIdentity=true \
+   --set enableAddOnDeploymentConfig=true \
+   --set hubDeployMode=AddOnTemplate
+cd -
+
+echo "Install managed-serviceaccount mca"
+clusteradm create clusterset spoke
+clusteradm clusterset set spoke --clusters ${c1},${c2},${c3}
+clusteradm clusterset bind spoke --namespace default
+kubectl apply -f env/placement.yaml || true
+kubectl apply -f env/mg-sa-cma-0.6.0.yaml || true
+
+echo "Install cluster-permission"
+cd /path/to/OCM/cluster-permission  # TODO: Replace here with your actual path.
+make install
+make deploy
+cd -
+
+echo "Install resource-usage-collect-addon"
+cd /path/to/addon-contrib/resource-usage-collect-addon  # TODO: Replace here with your actual path.
+IMAGE_NAME=zheshen/resource-usage-collect-addon:latest make deploy
```

Comment:
It seems the image name is hardcoded in addontemplate https://github.com/open-cluster-management-io/addon-contrib/blob/main/resource-usage-collect-addon/deploy/resources/addon-template.yaml#L53 , need to replace it with `example-addon-image`.

Reply:
Yes a new PR has been created to fix the image problem.
https://github.com/open-cluster-management-io/addon-contrib/pull/21